src/renderer/remote-loader.ts

```diff
@@ -14,19 +10,25 @@ import { ELECTRON_ORG, ELECTRON_REPO } from './constants';
 import { getTemplate } from './content';
 import { AppState } from './state';
 import { getReleaseChannel } from './versions';
+import { isKnownFile, isSupportedFile } from '../utils/editor-utils';
 
 export class RemoteLoader {
   constructor(private readonly appState: AppState) {
-    this.loadFiddleFromElectronExample.bind(this);
```

Comment:
i'm totally ok with this change but i find it perhaps little less readable and less idiomatic/typical than what was there previously

Reply:
What we had previously didn't actually work did it? The new function with `this` bound was discarded instead of remembered. In order to work it would  need to be `this.loadFiddleFromElectronExample = this.loadFiddleFromElectronExample.bind(this)`